gzip_cache/gzip_cache.py

```diff
@@ -77,12 +78,10 @@ def should_compress(filename, user_exclude_types):
     :param filename: A file name to check against
     :param user_exclude_types: User-supplied list of extensions to exclude
     '''
-    for extension in EXCLUDE_TYPES:
-        if filename.endswith(extension):
-            return False
-    for extension in user_exclude_types:
-        if filename.endswith(extension):
-            return False
+    if filename.endswith(EXCLUDE_TYPES):
+        return False
+    if filename.endswith(user_exclude_types):
+        return False
```

Comment:
`endswith` is picky about type, it really wants `tuple`s.

Reply:
Just noticed :) It's fixed now.